Beret armour now matches the security helmet

san7890 opened the pull request:
## About The Pull Request
This PR edits the beret's melee armor value down from 40 to 35, matching the same value of helmets.
Why It's Good For The Game

It makes absolutely no sense for a fancy cap made out of fabric to be STRONGER than a hard helmet.

## Changelog

:cl:
fix: Nanotrasen has now stopped including a big chunk of iron in their berets, making it a bit weaker to melee damage. Still just as fashionable, though.
/:cl:

san7890 committed:
soul: removed

decreases the beret's melee armor to be equivalent to the helmet

SuperNovaa41 commented:
This is balance not a fix

san7890 commented:
> This is balance not a fix

ok

optimumtact commented:
I'm not going to merge this until it removes all armour from berets, they're cloth hats

o0cyann0o commented:
lame and stinky

SuperNovaa41 commented:
> > I'm not going to merge this until it removes all armour from berets, they're cloth hats
> 
> ok

look what you did

san7890 commented:
> > > I'm not going to merge this until it removes all armour from berets, they're cloth hats
> > 
> > 
> > ok
> 
> look what you did

this is the result of a discussion i had with the man

san7890 committed:
armor is dead

orange man

qbmax32 commented:
what’s the issue if they have the same armor value as their “realistic” equivalents ie: sec beret has same armor as sec helmet? It’s just cosmetic at that point.

Fikou commented:
grief pr

optimumtact commented:
The issue is that it doesn't make sense and the fridge logic of it annoys me

o0cyann0o commented:
why did you change the description of the cc beret

san7890 committed:
hos beret

apparently it inherents it from parent (which is the HOS cap)

qbmax32 commented:
Also this removes armor on hos beret too which is dumb because it’s a unique head of staff item, it should keep its armor

san7890 commented:
> why did you change the description of the cc beret

because it makes sense

castawaynont commented:
Also remove armour off sec baseball caps and such

optimumtact commented on code/modules/clothing/head/jobs.dm:
```diff
@@ -242,12 +241,11 @@
 	greyscale_config = /datum/greyscale_config/beret_badge
 	greyscale_config_worn = /datum/greyscale_config/beret_badge/worn
 	greyscale_colors = "#972A2A#F2F2F2"
-	armor = list(MELEE = 40, BULLET = 30, LASER = 30, ENERGY = 40, BOMB = 25, BIO = 0, FIRE = 20, ACID = 50, WOUND = 4)
```
keep the armour, but reduce the values by half for all of it.

san7890 committed:
description

makes more sense

optimumtact commented on code/modules/clothing/head/jobs.dm:
```diff
@@ -242,12 +241,11 @@
 /obj/item/clothing/head/beret/sec
 	name = "security beret"
-	desc = "A robust beret with the security insignia emblazoned on it. Uses reinforced fabric to offer sufficient protection."
+	desc = "A robust (in looks) beret with the security insignia emblazoned on it."
```
undo this change

optimumtact commented on code/modules/clothing/head/jobs.dm:
```diff
@@ -359,7 +355,7 @@
 /obj/item/clothing/head/beret/centcom_formal
 	name = "\improper CentCom Formal Beret"
-	desc = "Sometimes, a compromise between fashion and defense needs to be made. Thanks to Central Command's most recent nano-fabric durability enhancements, this time, it's not the case."
+	desc = "Sometimes, a compromise between fashion and defense needs to be made. Thanks to Central Command's nano-fabric durability enhancements, it's not the case."
```
why are you even touching this?

san7890 replied:
It made sense at the time when everything was having armor removed since the old text was "most recent", because saying "most recent" when no other beret had armor didn't really feel correct to me personally. I rewrote it a little bit, but given the other recommended changes I'll probably revert it.

Nexofas commented:
> > why did you change the description of the cc beret
> 
> because it makes sense

Nanotech berets at 2500 makes sense. Berets are already inferior to helmets statwise. Leave berets alone so that some people can go fashionsouls.

optimumtact commented:
If you feel up to it, it would be nice to have a mid game tech tree unlock that unlocks a steel fibre thread that can be used to patch the beret to double it's armour again.

lemshake commented:
Kevlar is not a real material.

optimumtact commented:
>Nanotech berets at year 2500 makes sense. Berets are already inferior to helmets statwise. Leave berets alone so that some people can go fashionsouls.

askign people to make a tradeoff if they wish to look fashionable is acceptable, if the beret was wanted as the standard item, then the helmet should have been resprited.

optimumtact commented on code/modules/clothing/head/jobs.dm:
```diff
@@ -339,12 +336,11 @@
 	greyscale_config = /datum/greyscale_config/beret_badge
 	greyscale_config_worn = /datum/greyscale_config/beret_badge/worn
 	greyscale_colors = "#C5D4F3#ECF1F8"
-	armor = list(MELEE = 15, BULLET = 5, LASER = 15, ENERGY = 25, BOMB = 10, BIO = 0, FIRE = 30, ACID = 5, WOUND = 4)
```
as long as the values here are under the values provided by the sec badge, this is fine.

san7890 commented on code/modules/clothing/head/jobs.dm:
```diff
@@ -339,12 +336,11 @@
 
 /obj/item/clothing/head/beret/durathread
 	name = "durathread beret"
-	desc = "A beret made from durathread, its resilient fibres provide some protection to the wearer."
```
Do we want this to have armor? It doesn't make sense to say "some protection to the wearer" if we aren't going to give it armor.

TheBonded commented:
> The issue is that it doesn't make sense and the fridge logic of it annoys me

@optimumtact  then make sure to remove armour from the trenchcoats and winter coats too, they're also "just fabric." Obviously it's impossible to create a lightweight armour plating that you can put under fabric. Never been done in real life, and kevlar is a lie invented by security members to sell more AP 9mm magazines.

 Teleportation, magic glowy dust rocks, laser swords and time travel are all within the logic of the universe, but putting cloth over armour plates or using synthetic fibers woven into cloth are just too far fetched ideas to ever be realistic I guess.

optimumtact commented:
I"m really not interested in having a discussion with reddit.

san7890 committed:
orange man fixes part one

optimumtact commented:
Actually
>If you feel up to it, it would be nice to have a mid game tech tree unlock that unlocks a steel fibre thread that can be used to patch the beret to double it's armour again.

is a hard requirement, so if you dont' want to do the extra coding work, then just nerfing it to match the helmet is likely the best approach

let me know

san7890 committed:
descopifying

basically this should just match the original intent of the PR (which was to drop the beret armor by five points).

i spoke it over with orange man and (like above), armor WILL be nerfed on these items eventually until someone can do techtrees. That's about it.

tralezab commented:
this was silly, there armor back in line and some weird desc changes. ogey
